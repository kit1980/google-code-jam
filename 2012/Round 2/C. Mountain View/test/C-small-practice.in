30
6
2 3 4 5 6
4
4 4 4
4
3 4 4
4
4 3 4
2
2
10
2 3 4 5 6 7 8 9 10
7
3 7 4 6 6 7
6
6 6 5 5 6
5
4 5 5 5
7
7 3 5 7 7 7
7
2 4 6 5 6 7
10
3 10 10 10 10 10 10 10 10
7
6 4 5 5 6 7
6
5 3 4 5 6
7
2 7 5 7 7 7
5
2 4 5 5
10
10 10 10 10 10 10 10 10 10
5
3 4 5 5
7
3 3 6 6 6 7
6
2 6 6 5 6
6
2 6 4 5 6
7
2 6 6 6 6 7
10
10 6 6 5 6 8 9 10 10
8
3 3 4 8 6 8 8
5
3 5 4 5
8
3 7 6 5 6 7 8
5
4 3 4 5
7
2 3 5 5 6 7
10
10 4 8 7 6 7 8 9 10
10
10 9 8 7 6 7 8 9 10
