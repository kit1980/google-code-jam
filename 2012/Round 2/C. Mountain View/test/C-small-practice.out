Case #1: 0 0 0 0 0 0 
Case #2: 1000000000 0 0 1000000000 
Case #3: Impossible
Case #4: 1000000000 0 0 0 
Case #5: 0 0 
Case #6: 0 0 0 0 0 0 0 0 0 0 
Case #7: Impossible
Case #8: 1000000000 1 0 0 2 3 
Case #9: Impossible
Case #10: Impossible
Case #11: Impossible
Case #12: Impossible
Case #13: Impossible
Case #14: 3 0 1 1 1 0 
Case #15: Impossible
Case #16: Impossible
Case #17: 1000000000 999999999 999999999 999999999 0 0 0 0 0 1000000000 
Case #18: Impossible
Case #19: 0 0 1 0 0 1 0 
Case #20: 0 1000000000 1 0 0 0 
Case #21: 0 1000000000 0 0 0 0 
Case #22: 0 1 0 0 0 1 0 
Case #23: Impossible
Case #24: 0 0 2 3 0 1 0 0 
Case #25: Impossible
Case #26: Impossible
Case #27: 2 0 1 1 0 
Case #28: 0 1 1 0 1 1 0 
Case #29: Impossible
Case #30: 1000000000 0 0 3 0 9 18 23 27 30 
